redhat_upgrade_tool/commandline.py

```diff
@@ -45,6 +46,11 @@ def parse_args(gui=False):
     p.add_option('-f', '--force', action='store_true', default=False,
             help=_('continue even if the Preupgrade Assistant risk check'
                    'fails. Use it at your own risk.'))
+
+    p.add_option('--no-space-check', action='store_true', default=False,
```

Comment:
Just an aesthetics note. All the long options are without dashes separating the actual words like `--enablerepo, --nogpgcheck` except `--no-space-check`. This seems strange and inconsistent. I noticed it immediately.

But it's probably just me and my mild OCD.

Reply:
We discussed that earlier, but unfortunately both formats are already used (`--cleanup-post`).